### DISCUSSION TITLE
Xcode version 12.5.1

### DISCUSSION THREAD

**Discussion by chsaun** [appium/appium#15642] [Release]
I am getting an Appium error for iOS -- Unable to create driver for the given parametersMessage: A new session could not be created. Details: Appium's IosDriver does not support Xcode version 12.5.1. Apple has deprecated UIAutomation. Use the "XCUITest" automationName capability instead.

(I am using XCUITest for automationName capability.)

Appium desktop version:  1.21.0
XCode: 12.5.1 on Big Sur.  It was released on 6/21/2021

When will Appium support this version ?

thanks
Charlene

**Comment by KazuCocoa** [comment]
Could you attach the full server log please?
